src/navigation-instruction.js

```diff
@@ -228,16 +228,20 @@ export class NavigationInstruction {
   }
 
   _buildTitle(separator: string = ' | '): string {
-    let title = this.config.navModel.title || '';
+    let title = '';
     let childTitles = [];
 
+    if (this.config.navModel.title) {
```

Comment:
What if the title is the String '0', will it evaluate to false, or does it work as expected?